javatests/arcs/android/storage/database/DatabaseImplTest.kt

```diff
@@ -1881,6 +1954,17 @@ class DatabaseImplTest {
         }
     }
 
+    private fun readOrphanField(entityStorageKey: StorageKey): Boolean =
+        database.readableDatabase.rawQuery(
+            """
+                SELECT orphan
+                FROM entities
+                LEFT JOIN storage_keys ON entities.storage_key_id = storage_keys.id
+                WHERE storage_key = ?
+            """.trimIndent(),
+            arrayOf(entityStorageKey.toString()))
+        .forSingleResult { it.getNullableBoolean(0) } ?: false
```

Comment:
Paren goes on the next line for wrapped function calls.
```suggestion
            arrayOf(entityStorageKey.toString())
        ).forSingleResult { it.getNullableBoolean(0) } ?: false
```